### PR TITLE
cleaning up duplicate code in PositionalAudio and 'unsetting' filter

### PR DESCRIPTION
**Pull request by vincent-courtalon**
here a little enhancement to the audio_evolution #7485

### CODE REVIEW

File: src/audio/Audio.js [hunk 1]
```diff
@@ -131,14 +135,14 @@ THREE.Audio.prototype.stop = function () {
 
 THREE.Audio.prototype.connect = function () {
 
-	if ( this.filter !== undefined ) {
+	if ( this.filter != null ) {
```
Comment (mrdoob) [inline src/audio/Audio.js]: Minor thing, but better to use `!==` for consistency.

**Comment by vincent-courtalon** [comment]
no problems, was not sure which to use. you want to change it directly in the merge or you want me to update the pull request?

**Comment by mrdoob** [comment]
Updated the PR if possible.

**Comment by vincent-courtalon** [comment]
done!

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**